src/app/views.py

```diff
@@ -331,6 +343,63 @@ def licenseInformation(request, licenseId):
         )
 
 
+def licenseNamespaceInformation(request, licenseId):
+    """ View for license namespace request and archive request information
+    returns license_namespace_information.html template
+    """
+    if "archive_namespace_requests" in str(request.META.get('PATH_INFO')):
+        if not LicenseNamespace.objects.filter(archive='True').filter(id=licenseId).exists():
+            return render(request,
+            '404.html',{},status=404
+            )
+    else:
+        if not LicenseNamespace.objects.filter(archive='False').filter(id=licenseId).exists():
+            return render(request,
+            '404.html',{},status=404
+            )
+    licenseNamespaceRequest = LicenseNamespace.objects.get(id=licenseId)
+    context_dict = {}
+    licenseInformation = {}
+    licenseInformation['fullname'] = licenseNamespaceRequest.fullname
+    licenseInformation['shortIdentifier'] = licenseNamespaceRequest.shortIdentifier
+    licenseInformation['submissionDatetime'] = licenseNamespaceRequest.submissionDatetime
+    licenseInformation['userEmail'] = licenseNamespaceRequest.userEmail
+    licenseInformation['licenseAuthorName'] = licenseNamespaceRequest.licenseAuthorName
+    licenseInformation['archive'] = licenseNamespaceRequest.archive
+
+    licenseInformation['notes'] = licenseNamespaceRequest.notes
+    licenseInformation['namespace'] = licenseNamespaceRequest.namespace
+    licenseInformation['url'] = licenseNamespaceRequest.url
+    licenseInformation['description'] = licenseNamespaceRequest.description
+    licenseInformation['publiclyShared'] = licenseNamespaceRequest.publiclyShared
+    xmlString = licenseNamespaceRequest.xml
+    data = parseXmlString(xmlString)
```

Comment:
This function is defined in `utils`, I guess this is the reason the checks are failing